tools/experiments/experiments.js

```diff
@@ -73,6 +73,11 @@ const EXPERIMENTS = [
     name: 'Viewability APIs for amp-analytics',
     spec: 'https://github.com/ampproject/amphtml/issues/1297#issuecomment-197441289',
   },
+  {
+    id: 'amp-sticky-ad',
+    name: 'AMP Sticky Ad',
+    spec: '',
```

Comment:
There is: https://github.com/ampproject/amphtml/issues/2472

Speaking of which:
@zhouyx: Please add something like this to the description: "Initial implementation of #2472"

This links the pull request to the issue (In both directions) which can be very helpful for people to follow the work.